frontend/src/components/VCheckbox/VCheckbox.vue

```diff
@@ -4,7 +4,7 @@
     class="relative flex text-sm leading-5"
     :class="labelClasses"
   >
-    <div class="relative">
+    <span class="relative block">
```

Comment:
Only [phrasing content](https://developer.mozilla.org/en-US/docs/Web/HTML/Content_categories#phrasing_content) is allowed inside of `label`, so `div` should be replaced with a _phrasing_ `span`. (WebStorm was complaining about this)